manifests/cf-manifest/scripts/rotate-vars-store-secrets.rb

```diff
@@ -75,10 +78,13 @@ def rotate(manifest, vars_store,
            passwords: false,
            rsa: false,
            ssh: false,
+           vars_to_rotate: nil,
```

Comment:
should this be have a default value of "[]"? I expect we would get an error if we call `<nil>.include?'

Reply:
There is a `unless vars_to_rotate.nil?` just before.

being `vars_to_rotate=nil` means rotate all the vars.